docs/openapiv2/apidocs.swagger.json

```diff
@@ -1107,6 +1131,42 @@
         }
       }
     },
+    "Condition": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "title": "A unique name for the condition\nTODO(@jpadilla): Add validation rules"
```

Comment:
remove this TODO from here

Reply:
let's follow up on a separate pull request.

Reply:
this gets picked up automatically, i'll remove the actual TODO for now from this pull request